weather/src/components/WeatherContainer.js

```diff
@@ -78,10 +72,11 @@ class WeatherContainer extends Component {
           </Col>
         </Row>
           <Row>
-              <WeekForecast
+            {this.state.result.list ? <WeekForecast
```

Comment:
Doing an inline conditional but this could be brought to a function instead.

Reply:
Thanks again for the help Ron! Sorry I had that open, it's been hectic